django_spanner/introspection.py

```diff
@@ -76,7 +76,10 @@ def get_table_list(self, cursor):
         :rtype: list
         :returns: A list of table and view names in the current database.
         """
-        results = cursor.run_sql_in_snapshot(self.LIST_TABLE_SQL)
+        schema_name = self._get_schema_name(cursor)
+        results = cursor.run_sql_in_snapshot(
+            self.LIST_TABLE_SQL.format(schema_name=schema_name)
```

Comment:
I know it's not introduced by this PR, but if I understand it correctly, all of these methods generate a SQL statement with a literal in the string, instead of using a query parameter. That means that they are all potentially susceptible to SQL injection. Can we fix that for the queries that we are working on in this PR, and then do a follow-up PR for all other queries? (This should have a high priority)

Reply:
Fixed